### ISSUE TITLE
druapl css tweaks

### ISSUE THREAD

**Issue by jlyon** [proudcity/proud#72]

**Comment by jlyon** [comment]
- [ ] wysiwyg editor: styles

**Comment by aschmoe** [comment]
> JL: footer: why isn't Actions pane showing up (JL: it was yesterday before a db
> update)

Yeah not working when I pull the db locally.  Must be a features thing?

**Comment by aschmoe** [comment]
> footer actions: dropdowns should only be on click, not on hover (JL: not sure 
> why it's happening here but not on 
> http://pattern.getproudcity.com/?v=&page=agency-content)

Radix, removed this

**Comment by aschmoe** [comment]
> rm gray bar at the bottom of landing page w logged in (panel) - JL started trying 
> to do this in proud_landingpage.tpl.php in proud_panels

This was cause the footer region had .panel-footer which is a class used by bootsrap
